File: WindowsProgrammingVS/File Operations and IO/Form3.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace File_operations_in_dotnet
{
    public partial class Form3 : Form
    {
        public Form3()
        {
            InitializeComponent();
        }

        private void button2_Click(object sender, EventArgs e)
        {
            try
            {
                string userInput = textBox1.Text;
                int num = int.Parse(userInput);
                MessageBox.Show("You entered " + num.ToString());
            }
            catch (FormatException ex)
            {
                // raise error
                MessageBox.Show("Format error raised!");
            }
            catch (Exception ex)
            {

            }
            finally
            {
                // end the program
                MessageBox.Show("Code ended!");
            }
        }

        private void button3_Click(object sender, EventArgs e)
        {
            Form2 f = new Form2();
            this.Hide();
            f.Show();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            try
            {
                int a = 10;
                int b = 5;
                int c = a / b;
                // print the result
                MessageBox.Show(c.ToString());
            }
            catch (ArithmeticException ex)
            {
                // raise error
                // does not print default handler's message
            }
            finally
            {
                // end the program
                MessageBox.Show("Code ended!");
            }
        }

        private void button4_Click(object sender, EventArgs e)
        {
            Application.Exit();
        }
    }
}

File: Visual Basic.NET/Experiment 6/Form3.vb
Public Class Form3
    Private Sub button_zerodiv_Click(sender As Object, e As EventArgs) Handles button_zerodiv.Click
        Dim b As Integer = 0
        Try
            Dim a As Integer = Convert.ToInt32(textbox_num.Text)
            Dim c As Integer = a / b
            MessageBox.Show(c)
        Catch ex As FormatException
            MessageBox.Show("Format Exception raised!")
        Catch ex As OverflowException
            MessageBox.Show("Zero Division error!")
        Catch ex As ArithmeticException
            MessageBox.Show("Arithmetic exception raised!")
        Finally
            MessageBox.Show("Code ended!")
        End Try
    End Sub

    Private Sub button_next_Click(sender As Object, e As EventArgs) Handles button_next.Click
        Dim f As New Form4()
        Hide()
        f.Show()
    End Sub
End Class